letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -245,23 +245,41 @@ DeprecationBootstrap() {
 }
 
 
+DeterminePython2Version() {
+  DeterminePythonVersionBase $1 "$LE_PYTHON python2.7 python27 python2 python"
+}
+
+# TODO: use this to not upgrade if we already have python3 installed
```

Comment:
Yes. I thought having something like this might make it feasible, but on second thought we'd actually need all the other dependencies also, so unless we're switching entirely to checking for things before attempting to install them (which I am not proposing we do), we should probably just rebootstrap anyway.

Reply:
(done)